perf/config.go

```diff
@@ -29,6 +29,9 @@ type Events struct {
 	// output of perf list can be used.
 	Events [][]Event `json:"events"`
 
+	// List of uncore perf events to be measured.
+	UncoreEvents [][]Event `json:"uncore_events"`
```

Comment:
Since these events are completely distinct from "regular" perf events as far as defining CustomEvent and grouping, should they be in a separate struct?  I am fine with them being in the same configuration file, but it may make plumbing configuration easier, and lead to less confusion when writing the rules about what is possible.

Reply:
Can we stay with the same configuration?

Uncore is part of Perf so, in my opinion, it should be one configuration file.

Reply:
I'm fine with them being in the same file, but I think they should be nested separately to be clear that custom uncore events can only be listed in the UncoreEvent section.

Reply:
Is it looking good now?

Reply:
yes, thanks